src/pymordemos/elliptic2.py

```diff
@@ -6,18 +6,24 @@
 """Simple demonstration of solving the Poisson equation in 2D using pyMOR's builtin discretizations.
 
 Usage:
-    elliptic2.py [--fv] PROBLEM-NUMBER N
+    elliptic2.py [--fv] PROBLEM-NUMBER N NORM
 
 Arguments:
-    PROBLEM-NUMBER    {0,1}, selects the problem to solve
-    N                 Triangle count per direction
+    PROBLEM-NUMBER  {0,1}, selects the problem to solve
+    N               Triangle count per direction
+
+    NORM            h1: compute the h1-norm of the last snapshot.
+                    l2: compute the l2-norm of the last snapshot
+                    k: compute the energy norm of the last snapshot, where the energy-product is constructed
+                    with a randomly generated parameter instance with random seed integer k.
```

Comment:
Sorry, but why not just let the user specify the actual value? Specifying the seed seems weird, when it's only a single number we are talking about.

Reply:
You are absolutely right. I thought that problem 0 and 1 differ in terms of parameter space. I changed that now.